scripts/cangen/cangen/main.py

```diff
@@ -16,27 +16,41 @@ def parse():
     parser = argparse.ArgumentParser(description="DBC to C code generator")
     parser.add_argument("project", type=str, help="Name of the project")
     parser.add_argument(
```

Comment:
arguments look good. we shouldn't let the user pass both `--verbose` and `--cmake-log-level`, so make a mutually exclusive group for them https://docs.python.org/3/library/argparse.html#mutual-exclusion